integration-cli/docker_cli_plugins_test.go

```diff
@@ -32,7 +32,7 @@ func (s *DockerSuite) TestPluginBasicOps(c *check.C) {
 	id = strings.TrimSpace(id)
 	c.Assert(err, checker.IsNil)
 
-	out, _, err = dockerCmdWithError("plugin", "remove", pNameWithTag)
+	out, _, _ = dockerCmdWithError("plugin", "remove", pNameWithTag)
```

Comment:
Shouldn't this be `dockerCmd` instead ? Doesn't seem right to not care about error at all 😓